tests/unit/test_instance_cmds.py

```diff
@@ -1683,13 +1681,7 @@
       'test': 'in'},
      ASSOC_MOCK_FILE, OK],
 
-    ['Verify instance command associators, invalid instance name',
-     ['associators', 'TST_Blah.blah="abc"'],
-     {'stdout': '',
-      'rc': 0,
-      'test': 'lines'},
-     ASSOC_MOCK_FILE, OK],
-
+    # TODO: Starting with pywbem 0.15.0, this fails on some python version
```

Comment:
I suggest to investigate this until it is understood what the Python versions are where it fails, and how to specify the testcases as a result.